content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+    />
+    <Caption>If there are more than single options in a menu, have a user confirm their choices.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/40274682/228386685-4699374e-5f98-4dd4-bab8-f488ad0fff8a.png"
+    />
+    <Caption>
+      Never force users to submit their multi-select options by closing the dialog or auto-updating their choices upon
+      selection.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Click-to-dismiss behavior
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://user-images.githubusercontent.com/40274682/228388751-12c1f46e-aee7-4e03-8899-b273d31f7d1b.png"
+    />
+    <Caption>
+      Guide users to save or cancel their selections to dismiss Select panel to prevent confusion and data loss.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://user-images.githubusercontent.com/40274682/228388749-445619e6-91fa-4f9f-b300-6e1fe21c62a5.png"
+    />
+    <Caption>
+      Don’t allow a click-outside-to-dismiss behavior for Select panel. Users may be used to closing and submitting the
+      information. Doing so would result in a loss of their changes.{' '}
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Behavior
+
+### Opening
+
+<img
+  width="960"
+  alt="Mouse cursor hovering over an icon button and a focused icon button with the space and return keys visually below the icon button."
+  src="https://user-images.githubusercontent.com/40274682/228386681-e75b3bcd-ac0d-4e4a-afda-2a8dfc3a13a6.png"
+/>
+<Caption>
+  A Select panel trigger should be a semantic button. Mouse users can click and keyboard users can open with space or
+  enter.
+</Caption>
+
+<img
+  width="960"
+  alt="First select panel has focus applied to the search input with a blue 2px border. Second SelectPanel has no visible focus ring as focus is brought to the H1 of the dialog."
+  src="https://user-images.githubusercontent.com/40274682/228386679-02d288f4-7e1c-4a77-a654-03def368e266.png"
+/>
+<Caption>
+  If Select panel has an input, focus should be placed within the input. If not, focus should be placed in the next
+  logical place. That next logical place might just be the H1 of the dialog.
+</Caption>
+
+### Closing
+
+<Box display="grid" gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+  <div>
+    <CustomVideoPlayer
+      width="100%"
+      src="https://user-images.githubusercontent.com/40274682/228390156-dab0b413-3b90-4736-a0e7-8d5d30665c43.mp4"
+    />
+    <Caption>
+      Pressing esc, or selecting “cancel” should close the current dialog and return focus to the previous trigger. It
+      should not retain or submit any user input.
```

Comment:
Highlighting this as something that should be discussed with the above comments around dismissal.